packages/shared/src/hooks/utils/useVisualViewport.ts

```diff
@@ -0,0 +1,21 @@
+import { useState } from 'react';
+import { useEventListener } from '../useEventListener';
+
+interface VisualViewportResult {
+  width?: number;
+  height?: number;
+}
+
+const getVisualViewport = (): VisualViewportResult => ({
```

Comment:
Need to be able to ge the visual viewport. This is changed whenever on screen keyboard appears. We can leverage the height of the available window to size the comment input correctly in a way that you can see the action buttons on the bottom (otherwise those could be potentially hidden behind the on screen keyboard).